docs/writing-docs/mdx.md

```diff
@@ -160,6 +160,29 @@ To get a "documentation-only story", in your UI, define a `<Meta>` as you normal
 
 ![MDX docs only story](./mdx-documentation-only.png)
 
+## Linking to other stories and pages
+
+When writing MDX, you may want to provide links to other stories or documentation pages and sections. You can use the `path` query string.
+
+Considering a story with ID `some--id`, this redirects to the **Docs** tab of the story:
+```md
+[Go to specific documentation page](?path=/docs/some--id)
+```
+
+This redirects to the **Canvas** tab of the story:
+```md
+[Go to specific story canvas](?path=/story/some--id)
+```
+
+You can also use anchors to target a specific section of a page:
+```md
+[Go to the conclusion of the documentation page](?path=/docs/some--id#conclusion)
+```
+
+Note that when using the Controls addon, anchors are ignored by the Canvas tab since the URL gets rewritten to track the args values.
```

Comment:
@Dschungelabenteuer one thing about this. If you could adjust this to  :

```html
<div class="aside">
💡 <strong>Note:</> By applying this pattern with the Controls addon, all anchors will be ignored in Canvas based on how Storybook handles URLs to track the args values.
</div
```

Something like this. Let me know what you thing?

Reply:
Much better 👍

Reply:
Much better 👍